lib/acts_as_taggable_on_mongoid/models/concerns/tag_methods.rb

```diff
@@ -42,13 +42,12 @@ def create_tag(tag_definition, name)
 
           def as_8bit_ascii(string)
             string = string.to_s
-            if defined?(Encoding)
-              string.dup.force_encoding("BINARY")
-            else
-              # :nocov:
-              string.mb_chars
-              # :nocov:
-            end
+
```

Comment:
What happened here?

Reply:
I am not 100% sure.

From what I can remember `encoding("BINARY") was throwing exceptions and/or causing failing tests with Unicode characters.

I came to conclusion that this was code copied from another gem for an issue we've never had, and that the tests worked without it, so I would take the plunge and say that since it is failing now with Rails 5 that we didn't need it any more.